.changeset/real-kiwis-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+'ai': patch
+---
+
+fix (ai cli): fixes + bundle
```

Comment:
pr title and changeset should be more descriptive. this could be anything really, which does not help when scanning the code history